### ISSUE TITLE
List the contributors to an IEC

### ISSUE THREAD

**Issue by evanwolf** [openoakland/opendisclosure#385]
IECs may disclose some or all of the people, PACs, companies, and others who give them money. Let's divide each IEC's page in two columns; the left for their expenditures in an Oakland race (as we do now) and (new:) on the right for their receipts related to the race. 

When they don't report their contributors (dark money), we indicate that with text ("this organization does not report where their money comes from") and links to resources where you can find out more about this IEC and dark money in general (https://www.opensecrets.org/outsidespending/nonprof_summ.php). Or an action, like a form to join OpenSecrets.org or a similar transparency advocacy organization.

![blindfranklin](https://cloud.githubusercontent.com/assets/205622/6563539/144e3840-c65f-11e4-9253-8368a8a2b73d.jpg)
![darkmoneyhandout](https://cloud.githubusercontent.com/assets/205622/6563545/25eeb728-c65f-11e4-8797-04ab78e33d4a.jpg)

**Comment by chellrocks** [comment]
I hope that you don't take this the wrong way, but I don't think that we can do this. 1 - we have no relationship with opensecrets, sunlight foundation, or any of the existing non-profits.  2 - the idea of more/less of any single type of money in politics is inherently political.  3 - we will cut off a portion of the electorate who will no longer see us as unbiased factual organization.  Our goal is access to as much of the population as possible.  We defeat the purpose when we start linking to even mildly political organizations.

**Comment by mikeubell** [comment]
We already display the folks who contribute to committees that are required to file disclosure.   What more are you looking for?

I think CA requirements are more stringent than for the US.

**Comment by lla2105** [comment]
@chellrocks:  I agree with you Phebe

**Comment by tdooner** [comment]
It seems absolutely appropriate to me for us to somehow link our IECs to their representation on OpenSecrets or whatever other site has information about them.

It also seems fine to me to say tell our viewers that a particular IEC does not report its contribution sources. It might be "political" but it is not biased toward any candidate(s).

I don't see how the point 3 follows, because I believe we can represent this information neutrally to any particular candidate. If you mean that we will lose our neutrality on the debate of "should there be dark money in politics?" then that is a different question -- but I would argue that us taking a stand against dark money is the whole point of the project, so we've already come out strongly in regards to that question.

**Comment by evanwolf** [comment]
From back at our summit in November... 
https://opendisclosure.hackpad.com/Our-Ethics-Non-partisan-vs.-Neutral-jtZ6ENRy8Sf 

# Our Ethics: Non-partisan vs. Neutral

We believe campaign finance disclosure sites have a duty to be non-partisan, not taking sides on whether a proposed law should pass or which candidates should win an election. 

Mostly. 

Let's explore the exceptions. 

## We should engage when faced with...

### Better disclosure measures

We should support laws and regulations that improve our ability to understand the flow of money, influence, and power in local politics. These include open data laws, laws that improve the freshness, completeness, and accuracy of campaign finance data. 

### Obfuscation

We should actively oppose the misuse of campaign finance disclosure systems to hide or mislead the actions of moneyed participants. 

### Candidates hostile to transparency

It's worth communicating to our own constituents when candidates or others actively oppose our work, or challenge our core values: sunshine, open data, citizen engagement. 

## We are not neutral

In addition to pride in our civic hacking, we stand for other powerful ideas.

- *Transparency.* This leads to accountability by electeds and engagement by residents. We are for more transparency, subject to  reasonable concerns for safety and privacy. 
- *Open Data.* Transparency works when data is free, open to the public, and shared. We want more and better data from and about those who put money into politics and those who spend it. 
- *Money Matters.* Money represents the will of contributors and influences how voters vote. Transparency about contributions and spending can keep elections cleaner, fairer, and help voters make better decisions. 

- *Civic Hacking.* Constituents have a vital role and duty to help their local governments innovate. We support efforts to enable volunteer engagement in the work of government.  

== Future Advocacy

Potential advocacy campaigns... 

- *Dark Money.* Local and state versions of the proposed Disclosure Act, increasing transparency on local money  from IECs and companies. 
- *Open Data.* Candidate Open Data Pledge campaigns. 
- *Continuous Reporting.* Local rules that require transactions be reported within two weeks; within one week 90 days before and after election day for $1000+ campaigns. 
- *Electronic Filing Everywhere.* Require electronic filing in more cities, counties, districts.

**Comment by stochasticTreat** [comment]
Yeah, I agree, we are providing ways to show how money is distributed
across different dimensions of contributors and candidates -- we show how
it is distributed across geographic regions, and contributor type. The
amount of dark money--that coming from undisclosed sources--is just another
dimension, and a very pertinent one moreover.
I do not think it innately favors one side or the other.
And most importantly, I strongly believe that reporting how much money is
from undisclosed sources will drive interest in our sight to a much greater
extent than it might ever induce people to believe we are unbiased or
factual.
So far as linking, it might suffice to provide two links: one to a more
formal, unbiased discussion of IECs (ie wikipedia) and another to whatever
source we can find that might fill in these blanks, along with a note
indicating we are not connected to the linked to website(s) but that they
are the only source(s) we know of that might be able to fill in the blanks
in question. We could even encourage users to suggest other sources.

**Comment by stochasticTreat** [comment]
*than it might ever induce people to believe we are **not** unbiased or
factual.
:)

**Comment by evanwolf** [comment]
@chellrocks I appreciate your caution. We should be careful not to be (and not to appear) for or against a political party or any candidate or measure. 

That said, we're a "good government" organization. So we care about protecting the democratic process. 

Even that is too broad, I think. So, we care about protecting the openness and transparency that allows America to trust our democratic process. 

This means we don't care if candidates lie about their campaign promises, or even their campaign finances. We just care that the process for keeping their disclosures fresh and accurate is trustworthy.

**Comment by mikeubell** [comment]
My reading of http://www.fppc.ca.gov/manuals/Manual6_0214.pdf <http://www.fppc.ca.gov/manuals/Manual6_0214.pdf> leads me to believe that there no legal dark money in California campaigns.  

We start with the definition of those who must file:
Individuals or entities using their own funds to make independent expenditures on candidates or measures should use this manual. Individuals or entities, including corporations, firms, businesses, or proprietorships, making independent expenditures of $1,000 or more in a calendar year qualify as a committee under the Act and must file reports of the independent expenditures.

Then:
Each independent expenditure of $1,000 or more on behalf of a candidate or ballot measure triggers the filing of campaign reports: Form 462, Form 465, Form 461 and Form 496.

This means that they must report money in and money out.

If I am misinterpreting the law, can someone tell me where the loop hole is?

Mike

**Comment by stochasticTreat** [comment]
From what I understand, 'Social Welfare' organizations (ex Crossroads,
Planned Parenthood, aka 501(c)4 organizations) can donate money and publish
politically biased content, but they do not have to disclose their own
lists of contributors. Thus, the money coming from their contributors might
be considered 'dark money'.
See:
https://upload.wikimedia.org/wikipedia/commons/4/4d/Campaign_finance_web_final.png
And/or:
http://en.wikipedia.org/wiki/American_Crossroads
and/or:
https://en.wikipedia.org/wiki/501%28c%29_organization#501.28c.29.284.29

**Comment by evanwolf** [comment]
@stochasticTreat Good point. The US Chamber of Commerce and other large 501c6 trade associations have the same disclosure obligations: report your political spending; reporting your own contributors is optional. The NRA's political arm is basically under the same rules. 

For our purposes, our aim is to help people understand more than the accounting; to understand the patterns of political influence. 

You're bringing up a really important point. We don't particularly care about the intentions of those who channel dark moneys into a local election. That value judgement is up to our users. Our role is to reveal that some money's sources are harder to know than others; and so the intentions and patterns behind that money may be opaque. Where money is speech, and the speech is political, the public should know the speaker. 

If the AARP spent money to support a measure on an Oakland ballot, they wouldn't list all their members. That would be impractical. But they publish an annual report that lists their sources of income, and policy platforms that guide their political spend. They are, to a degree, knowable. 

Other sources of IEC funds are more intentionally hidden. And their purposes may be impossible to discover before voting starts. 

So let's do what's possible. Report what's on the IEC's disclosure forms. Add in descriptive data provided by watchdog organizations. Sprinkle in local journalism and community discussion. It won't be complete or perfect. We can just hope it's closer to a complete picture.

**Comment by tdooner** [comment]
As was mentioned at the SF campaign finance summit yesterday, most people consider 501(c)4 donations to be the biggest source of "Dark Money" in California races.

* 501(c)4's can only spend up to 50% of their money on political activism, but in practice that just means that they have to give 50.0000001% of their money to another 501(c)4.
* 501(c)4 contributions are not disclosed, except confidentially to the IRS
* 501(c)4 contributions are tax deductible for donors
* However, they are barred from using certain magic words in their advertisements - "vote for/against" is the most common example.

Not to say that other sources of dark money don't exist, but California has good disclosure laws compared to what exists in other states and at the federal level.

**Comment by mikeubell** [comment]
Thanks for this input.  One point.  501(c)4 contributions are not deductible as charitable donations but only as business expenses, except that they are not deductible if they go to Lobbying or Political Expenditures: http://www.irs.gov/Charities-&-Non-Profits/Other-Non-Profits/Donations-to-Section-501%28c%29%284%29-Organizations <http://www.irs.gov/Charities-&-Non-Profits/Other-Non-Profits/Donations-to-Section-501(c)(4)-Organizations>.
It would probably be easy for an org to hide this distinction as I doubt there is much auditing in this area.

Mike
